client/components/domains/mixins/domain-suggestion-mixin.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal Dependencies
+ **/
+import cartItems from 'lib/cart-values/cart-items';
+
+export default {
+	willBundle( withPlansOnly, selectedSite, cart, suggestion ) {
```

Comment:
I'm wondering why you're passing `withPlansOnly` here, instead of pulling it directly from `abtest`? I understand that pulling from a global "state" like this is kind of a code smell, but in the end, we'll be removing the need for this variable completely when we transition to DWPO permanently for all users so it shouldn't be an issue (and that's how we implement other abtest-specific logic, from what I've seen).